TROUBLESHOOT.md

```diff
@@ -0,0 +1,34 @@
+# Troubleshooting Origami Build Tools
+
+Different common issues encountered when using OBT organized by the different tasks.
+
+## `install`
+
+### Installing Ruby
+
+* Macs typically ship with Ruby by default, but:
+	- If you get a `Gem::FilePermissionError` error, you need to [install a ruby version manager](http://stackoverflow.com/questions/19579392/installing-gem-fails-with-permissions-error).
+	- To [install rbenv](https://github.com/sstephenson/rbenv#homebrew-on-mac-os-x) you first need to [install homebrew](http://brew.sh/).
+	- Once you've installed rbenv you need to [install ruby](https://github.com/sstephenson/rbenv/#installing-ruby-versions).
+		- Note: rbenv needs a bit of post-install setup. If you install with homebrew, also do steps 2 and 3 of the [Basic GitHub Checkout](https://github.com/sstephenson/rbenv/#basic-github-checkout) section.
+
+### Error `Unable to download data from https://rubygems.org/`
+
+If you receive an error specifying `Unable to download data from https://rubygems.org/ - SSL_connect returned=1 errno=0 state=SSLv3 read server certificate B: certificate verify failed (https://api.rubygems.org/specs.4.8.gz)` you'll need to manually update your gem package using the directions in [this gist](https://gist.github.com/luislavena/f064211759ee0f806c88).
+
+### Error `While executing gem ... (Gem::FilePermissionError)`
+
+When installing Sass and/or ScssLint, depending on how you have your permissions configured in your computer, you might get this error. You will need to run the command manual prefixing sudo to it. To install the supported version you need to run the following command (substituting the version number by the one mentioned [here](https://github.com/Financial-Times/origami-build-tools/#install)):
+
+```bash
+sudo gem install scss-lint -v 0.34.0
+sudo gem install sass -v 3.4.0
+```
+
+### Warning `Origami registry is not configured in a .bowerrc file`
+
+The install task runs `bower install`, which needs to check the Origami registry apart from the bower one. That's why you need to configure a `.bowerrc` file. You can do this in your home directory by running the following command:
```

Comment:
> which needs to check the Origami registry apart from the bower one.

I find this ambiguous. Maybe clearer to say something like "Origami modules are listed on Origami's custom Bower registry, in opposition to the global, public one".

> That's why you need to configure a `.bowerrc` file. You can do this in your home directory by running the following command:

Too personal? What do you think of: "Custom registries can be added to a [`.bowerrc`](http://bower.io/docs/config/) configuration file located in your home directory. To create a `.bowerrc` configuration file now, run:…"